bundles/org.eclipse.ui.monitoring/src/org/eclipse/ui/internal/monitoring/preferences/MonitoringPreferenceInitializer.java

```diff
@@ -30,7 +30,7 @@ public void initializeDefaultPreferences() {
 
 		store.setDefault(PreferenceConstants.MONITORING_ENABLED, false);
 		store.setDefault(PreferenceConstants.LONG_EVENT_WARNING_THRESHOLD_MILLIS, 500); // 0.5 sec
-		store.setDefault(PreferenceConstants.LONG_EVENT_ERROR_THRESHOLD_MILLIS, 1000); // 2 sec
+		store.setDefault(PreferenceConstants.LONG_EVENT_ERROR_THRESHOLD_MILLIS, 100_000); // 100 sec
```

Comment:
So we still pop up a warning on freeze, except we allow freezes to be much longer before we pop up an info?

I think that's an okay option, especially since it is relatively easy to implement compared to it's revenue (i.e. you are not always annoyed by a popup on smaller freezes)

Reply:
We will still **log** a warning, but **activate** the error view only for 100s lags (i.e. almost never)